Formula/qt5.rb

```diff
@@ -121,11 +121,16 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -system-proxies
     ]
 
     args << "-nomake" << "examples" if build.without? "examples"
 
-    args << "-plugin-sql-mysql" if build.with? "mysql"
+    if build.with? "mysql"
+      args << "-plugin-sql-mysql"
+      inreplace "qtbase/configure", /(QT_LFLAGS_MYSQL_R|QT_LFLAGS_MYSQL)=\`(.*)\`/, "\\1=\`\\2 | sed \"s/-lssl -lcrypto//\"\`"
```

Comment:
no, this only removes openssl from build time(build time for mysql plugin), libmysqlclient still need openssl at runtime.